x-pack/plugins/lens/public/drag_drop/providers.tsx

```diff
@@ -171,7 +171,11 @@ export function ReorderProvider({
       <EuiPortal>
         <EuiScreenReaderOnly>
           <div>
-            <p id="lnsDragDrop-reorderAnnouncement" aria-live="assertive" aria-atomic={true}>
+            <p
+              id={`lnsDragDrop-reorderAnnouncement-${id}`}
```

Comment:
2 things here:
* Why do we need an id on this element? AFAICT it's not referenced anywhere (and if it is, we have to adjust that place as well)
* The test is failing because the id is the group id, so if there are multiple layers there will be duplicates. Probably fixable by incorporating the layer id into the reorder provider id here: https://github.com/elastic/kibana/blob/898385c2e230e665dc9e5ca9ee231f446213ade1/x-pack%2Fplugins%2Flens%2Fpublic%2Feditor_frame_service%2Feditor_frame%2Fconfig_panel%2Flayer_panel.tsx#L225

Reply:
Looking at the debug page for the failure, there's effectively a duplication of `div`s, hence the id duplication.
Removing the `id` from the `div` will certainly fix the test, but there's still a problem with having twice as much divs with the same content on the page. I'll try to fix this as well.